langchain4j-core/src/main/java/dev/langchain4j/model/chat/ImageProcessingModel.java

```diff
@@ -0,0 +1,8 @@
+package dev.langchain4j.model.chat;
+
+import dev.langchain4j.data.image.Image;
+import dev.langchain4j.model.output.Response;
+
+public interface ImageProcessingModel {
```

Comment:
Yes, my intention about the name was to be more generic in case of https://platform.openai.com/docs/api-reference/images/createEdit, indeed this is supported by DALL·E 2 only. I'm fine with `ImageModel` and to be honest I cannot remember why I've put it into `chat` 😄, `image` package makes more sense 😄

Reply:
done